src/core/exec-invoke.c

```diff
@@ -2244,14 +2244,42 @@ static int build_pass_environment(const ExecContext *c, char ***ret) {
         return 0;
 }
 
-_noreturn_ static void bpffs_helper(int parent_fd) {
+_noreturn_ static void bpffs_helper(int parent_fd,
+                                    uint64_t delegate_cmds,
+                                    uint64_t delegate_maps,
+                                    uint64_t delegate_progs,
+                                    uint64_t delegate_attachs) {
         _cleanup_close_ int fs_fd = -EBADF, mnt_fd = -EBADF;
+        char number[16 + 2 + 1];
+        const char *s;
         int r;
 
         fs_fd = receive_one_fd(parent_fd, 0);
         if (fs_fd < 0)
                 _exit(EXIT_FAILURE);
 
+        if (delegate_cmds != ~0ULL) {
```

Comment:
`!= UINT64_MAX` ?

Reply:
As you prefer, that was how the corresponding kernel code generates that number:

https://elixir.bootlin.com/linux/v6.14.8/source/kernel/bpf/inode.c#L917